tests/review-child-inputs.test.sh

```
@@ -658,9 +664,31 @@ git -C "$SCOPE_REPO" commit -qm 'test: create initial review change'
   PR_NUMBER=73
   REVIEW_REPO_SLUG="$SCOPE_EXPECTED_REPO_SLUG"
   [[ "$REVIEW_REPO_SLUG" =~ ^[A-Za-z0-9_.-]+/[A-Za-z0-9_.-]+$ ]]
+  # #440 — this fence is the ONLY writer of the reviewed-base identity, and the
+  # anchor/audit consumers live 48+ fences downstream in different harness
+  # shells. Give it the run dir and the audit emitter a real run has, so the
+  # carrier write is executed rather than asserted about.
+  UBERDEV_REVIEW_PLUGIN_ROOT="$ROOT/plugins/uberdev"
+  RESEARCH_DIR_ABS="$TMP/scope-run"
```

Shellcheck (0.11.0)

[info] Modification of RESEARCH_DIR_ABS is local (to subshell caused by (..) group).

(SC2030)